tools/createGenesis/generateGenesis.ts

```diff
@@ -29,7 +29,7 @@ import {ProxyAdmin} from "../../typechain-types";
 import {Addressable} from "ethers";
 import "../../deployment/helpers/utils";
 
-const deployParameters = require("./deploy_parameters.json");
+const genesisParameters = require("./genesis_parameters.json");
```

Comment:
this parameters shoudl match the "deploy_parameters" ones, if you do this you shoudl assert against them, i would not make this change